packages/pipeline/migrations/1543540108767-CreateNftTrades.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const nftTrades = new Table({
+    name: 'raw.nonfungible_dot_com',
```

Comment:
Can we call the table `non_fungible_dot_com_trades` instead of just `non_fungible_dot_com`? This will more accurately describe what the table holds (we might get other data from nonfungible.com in the future) and is more aligned with our other table names.